Doc/library/datetime.rst

```diff
@@ -2432,7 +2432,7 @@ Class attributes:
 
 :class:`date`, :class:`.datetime`, and :class:`.time` objects all support a
 ``strftime(format)`` method, to create a string representing the time under the
-control of an explicit format string.
+control of an explicit format string. The methods only accept ASCII digits.
```

Comment:
Pleaee mention that a ValueError js raised otherwise (or locate where we mention ValueErrors for that and indicate that non-ASCII digits may be the cause of the exception). If there js no mention of ValueError, mention it more generally like "A ValueError is raised if the input cannot be parsed"